diff -rupN a2jmidid-6.old/wscript a2jmidid-6/wscript
--- a2jmidid-6.old/wscript	2009-12-06 19:53:25.000000000 -0500
+++ a2jmidid-6/wscript	2010-05-19 13:13:20.000000000 -0400
@@ -130,6 +130,7 @@ def build(bld):
     prog.includes = '.' # make waf dependency tracking work
     prog.target = 'a2jmidid'
     prog.uselib = 'ALSA JACK DBUS-1'
+    prog.env.append_value("LINKFLAGS", "-ldl")
 
     prog = bld.create_obj('cc', 'program')
     prog.source = 'a2jmidi_bridge.c'
