diff -rupN a2jmidid-8.old/wscript a2jmidid-8/wscript
--- a2jmidid-8.old/wscript	2012-07-04 21:14:23.000000000 -0400
+++ a2jmidid-8/wscript	2012-07-09 22:38:36.474015629 -0400
@@ -67,6 +67,7 @@ def configure(conf):
         conf.env['DBUS_ENABLED'] = False
 
     conf.env['LIB_DL'] = ['dl']
+    conf.env['LIB_PTHREAD'] = ['pthread']
 
     #conf.check_header('expat.h', mandatory=True)
     #conf.env['LIB_EXPAT'] = ['expat']
@@ -152,7 +153,7 @@ def build(bld):
 
     prog.includes = '.' # make waf dependency tracking work
     prog.target = 'a2jmidid'
-    prog.uselib = 'ALSA JACK DL'
+    prog.uselib = 'ALSA JACK DL PTHREAD'
     if bld.env()['DBUS_ENABLED']:
         prog.uselib += " DBUS-1"
     prog = bld.create_obj('cc', 'program')
